### PR TITLE
ChakraFull/ChakraCore Intl.js bytecode parity

### PR DESCRIPTION
**Pull request by tcare**
This is a precursor to bytecode verification unit tests, which requires ChakraFull to produce the same library bytecode as ChakraCore for Intl.js.
- Removed an implicit fscrNoAsmJs parser flag in ScriptContext when entering from JSRT. We don't need this flag on the initial entry, so it can be removed from the JSRT entrypoint for library bytecode generation.
- Zero out bytecode version when generating library bytecode.

### CODE REVIEW

**Comment by tcare** [comment]
@curtisman @MikeHolman please review

File: lib/Runtime/ByteCode/ByteCodeSerializer.cpp [hunk 1]
```diff
@@ -418,33 +419,57 @@ class ByteCodeBufferBuilder
         }
 
         // library alaways use the release versioning scheme
```
Comment (dilijev) [inline lib/Runtime/ByteCode/ByteCodeSerializer.cpp]: Update comment

**Comment by dilijev** [comment]
Please also regen Intl.js.nojit.bc.32b.h (you only regen'd Intl.js.nojit.bc.64b.h for NoJIT)

Also just to be sure everything is good, when you're ready, please run the NoJIT tests: https://github.com/Microsoft/ChakraCore/wiki/Jenkins-Build-Triggers#disablejit-builds

**Comment by dilijev** [comment]
LGTM

**Comment by tcare** [comment]
@dotnet-bot test nojit tests please

**Comment by dilijev** [comment]
@dotnet-bot help

**Comment by tcare** [comment]
Ping...

I couldn't repro the disablejit problems, trying again.

@dotnet-bot test nojit tests please

**Comment by dilijev** [comment]
@tcare looks good now. Not sure what happened before...

File: lib/Runtime/Base/ScriptContext.cpp [hunk 2]
```diff
@@ -1700,7 +1700,7 @@ if (!sourceList)
 
         if((loadScriptFlag & LoadScriptFlag_isByteCodeBufferForLibrary) == LoadScriptFlag_isByteCodeBufferForLibrary)
         {
-            grfscr |= (fscrNoAsmJs | fscrNoPreJit);
```
Comment (abchatra) [inline lib/Runtime/Base/ScriptContext.cpp]: Why is this change required?

Comment (tcare) [inline lib/Runtime/Base/ScriptContext.cpp]: grfscr gets encoded into library bytecode as part of function body. We don't set it in JsHost, so we don't need to set it in ch either (checked with @MikeHolman)

**Comment by abchatra** [comment]
:shipit:
